agent/bench-scripts/pbench-fio

```diff
@@ -12,6 +12,7 @@ pbench_bin="`cd ${script_path}/..; /bin/pwd`"
 
 benchmark_rpm=$script_name
 benchmark="fio"
+fio_server_port=8765
```

Comment:
Agree, but making it a parameter is a much bigger PR. I prefer to keep it as a var from now. Just to make the code cleaner. If there is a need for a paramater in the future It will make things easier to refactor.